tests/test_id_age_rating/test_excel_multi_sheets.py

```diff
@@ -75,15 +74,10 @@ def compare_custom_date(read_data, custom_data):
         assert len(read_data) == len(self.SHEETS)
         assert all(compare_custom_date(read_data[sheet], custom_data[sheet]) for sheet in self.SHEETS)
 
-        out_dn2.write(read_data)  # TODO: Fix this. It does not work to write custom data to excel
+        out_dn2.write(read_data)
         assert len(out_dn2.read()) == len(self.SHEETS)
         assert all(compare_custom_date(custom_data[sheet], out_dn2.read()[sheet]) for sheet in self.SHEETS)
 
-        out_dn2.write(None)
-        assert isinstance(out_dn2.read(), list)
-        assert len(out_dn2.read()) == 0
-        assert all(compare_custom_date(custom_data[sheet], out_dn2.read()[sheet]) for sheet in self.SHEETS)
```

Comment:
DO we know why this does not work? I believe it is something we want to work, no?

Reply:
No, this is copy pasted from other tests.
On multisheet excel, we can only write data as a dict with keys are sheet name.

Reply:
How do you write an empty file, then?
`out_dn2.write({})` like that?

I would prefer to make it possible to write None data regardless of the datanode storage type or exposed type. I don't know if it is feasible.

Reply:
yes, we can have a test that write empty data in a form of a dictionary

Reply:
@jrobinAV We don't have tests that write empty on taipy. What do you think about adding the tests on taipy instead of here?

Reply:
Makes a lot of sense to me.